### PR TITLE
JN-488 cleaning up enrollee view

### PR DESCRIPTION
**Pull request by devonbush**
#### DESCRIPTION

image

As a precursor to implementing actionable tasks, we needed to clean up the presentation on the enrollee admin view.  This brings it closer ot the mockups at https://dspuxteam.invisionapp.com/console/Juniper-Nav-Test-2-clkh8yvnz61gv01c49vpe85p3/cll101enq3ijk017yexgv62ix/play

This also refactors the Enrollee loading components, factoring out a "useRoutedEnrollee" hook to simplify the as-rendered component hierarchy

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

1. go to `https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/participants/OHSALK`
2. confirm it matches the screenshot above
3. click around, confirm enrollee menu still works

### CODE REVIEW

File: ui-admin/src/study/participants/enrolleeView/EnrolleeView.tsx [hunk 1]
```diff
@@ -70,89 +85,99 @@ export default function EnrolleeView({ enrollee, studyEnvContext, onUpdate }:
...
-        </div>
       </div>
     </div>
     <div className="row mt-2">
       <div className="col-12">
         <div className="d-flex">
-          <div style={{ minWidth: '280', maxWidth: '280px' }}>
-            <ul className="list-group">
-              <li className="list-group-item">
+          <div style={{ minWidth: '290', maxWidth: '290px' }}>
```
Comment (MatthewBemis) [inline ui-admin/src/study/participants/enrolleeView/EnrolleeView.tsx]:
```suggestion
          <div style={{ minWidth: '290px', maxWidth: '290px' }}>
```

`minWidth` is being ignored without a unit, which causes the width of the sidebar to suddenly change as sections are collapsed

File: ui-admin/src/App.tsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ function PageFrame() {
   return (
     <div className="d-flex">
       <AdminSidebar/>
-      <div className="flex-grow-1 d-flex flex-column" style={{ backgroundColor: '#ededed' }}>
+      <div className="flex-grow-1 d-flex flex-column" style={{ backgroundColor: '#fff' }}>
```
Comment (MatthewBemis) [inline ui-admin/src/App.tsx:84]: I honestly don't have a strong opinion on this in terms of styling, but this change is pretty far-reaching and causes other parts of the admin app to look less defined. In some cases, it's a bit harder to visually associate actions with their context as a result. Just want to call it out to make sure that this is intended. For example:
Screenshot 2023-09-28 at 4 28 53 PM

**Review by MatthewBemis — APPROVED** [review]
Two small comments.

File: ui-admin/src/App.tsx (hunk 2 above)
Comment (devonbush) [reply ui-admin/src/App.tsx:84]: ooh, that's a good catch, I hadn't tried that particular page -- I'll see if I can clean that up

**Review by denniscunningham — APPROVED** [review]
I gave this a good look, FWIW, which may not be much of a review at this point.
Good with this when Matt approves.

**Pull request merged**